de.tubs.variantsync.core/src/de/tubs/variantsync/core/patch/DeltaFactoryManager.java

```diff
@@ -63,5 +73,53 @@ public IDeltaFactory getFactoryByFile(IFile file) throws NoSuchExtensionExceptio
 	public static IDeltaFactory getDefaultFactory() {
 		return DefaultDeltaFactory.getInstance();
 	}
+	
+	@Override
+	public synchronized List<IDeltaFactory> getExtensions() {
+		if (extensionLoader != null) {
+			synchronized (extensions) {
+				if (extensionLoader != null) {
+					extensionLoader.loadProviders(this);
+					extensionLoader = null;
+				}
+			}
+		}
+		return Collections.unmodifiableList(extensions);
+		}
```

Comment:
The indents look all over the place, tab size = 4 and proper indention should be enforced by the style config @pmbittner